src/features/auth/lib/clearEntityRecordStorages.ts

```diff
@@ -1,5 +1,5 @@
-import { getDefaultMediaFilesCleaner } from '@app/entities/activity/lib/services/mediaFilesCleanerInstance';
-import { getDefaultStorageInstanceManager } from '@app/shared/lib/storages/storageInstanceManagerInstance';
+import { getDefaultMediaFilesCleaner } from '@entities/activity/lib/services/mediaFilesCleanerInstance.ts';
+import { getDefaultStorageInstanceManager } from '@shared/lib/storages/storageInstanceManagerInstance.ts';
```

Comment:
Is there a reason to add the `.ts` extension to these import paths?

Same thing [here](https://github.com/ChildMindInstitute/mindlogger-app-refactor/pull/859/commits/4af9ecb54895c8e6b51e12b5008a584a06359ec6#diff-671e9b4b9ec2d18a12b3dd87d343a500e580f98c83de6fa62cbf79e8942f2757R1), [here](https://github.com/ChildMindInstitute/mindlogger-app-refactor/pull/859/commits/4af9ecb54895c8e6b51e12b5008a584a06359ec6#diff-a83f923d379c68ecb76a5b0f9307a8ab4b3971a6c8e2e15f8b56f95121ba0809R1-R3), [here](https://github.com/ChildMindInstitute/mindlogger-app-refactor/pull/859/commits/4af9ecb54895c8e6b51e12b5008a584a06359ec6#diff-2b00f31120d77882c5b15cd11083e8e063f5e3339dc813d7674034e69fa3fb24R31), and [here](https://github.com/ChildMindInstitute/mindlogger-app-refactor/pull/859/commits/4af9ecb54895c8e6b51e12b5008a584a06359ec6#diff-46a7e9d5ae997ecd9b3d9fe46b52910b3fbc1f65ffe8e3f5e222ab488faaef10R19).

Reply:
I'm actually not sure. Both VScode and WebStorm did that in separate instances. I'm not sure why that's the case. But I can manually remove those extensions from important statements (it should be easy to find them all).

Reply:
Actually we're using `eslint-plugin-import`, so we can just add a `"import/extensions": "never"` rule 💁‍♀️ 
 https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/extensions.md

Reply:
Turns out that rules doesn't support auto-fix 😵 I guess I'll just manually remove them.